tips-for-getting-your-next-developer-job.rst

```diff
@@ -0,0 +1,183 @@
+.. post:: February 13, 2019
+   :tags: advertising, sustainability, hiring
+   :author: David
+   :location: SAN
+
+
+Tips For Getting Your Next Developer Job
+========================================
+
+Over the last month, the Read the Docs team conducted 30-40 customer development interviews
+with hiring managers and recruiters
+at companies ranging from 5-person companies to the biggest names in tech.
+We wanted to learn more about hiring processes at various companies
+with the ultimate goal of building `a product to help companies find developers`_.
+
+Last time, we covered some :doc:`tips for hiring managers <lessons-from-hiring-manager-interviews>`
+based on what companies told us they were doing.
+This time, we put together **tips for candidates looking for their next job**
+based on insights we heard from hiring managers.
+
+Rather than covering how to crack developer interviews which is its own challenge
+or how to gain the necessary technical skills,
+these tips will focus on getting your foot in the door. Specifically:
```

Comment:
Bit of a run on sentence. Not sure the interview part is even needed.

Reply:
I think cutting "which is its own challenge" does improve the sentence overall but cutting the unnecessary parts.